### ISSUE TITLE
Fix HTML validation error: bad value "manifest"

### ISSUE THREAD

**Issue by timur-friedman** [EdgeNet-project/edgenet-legacy-2024#80]
We get the following HTML validation error:

> Bad value manifest for attribute rel on element link: The string manifest is not a registered keyword or absolute URL. Not an absolute IRI.

 
On the following code in index.html:

https://github.com/EdgeNet-project/edgenet/blob/92c44bb82dbecff5def9ea98f4f52fe4ca411427/website/index.html#L28

**Comment by bsenel** [comment]
I take advantage of the W3C service at https://validator.w3.org/ for the validation and didn't encounter this error. Can you please share the tool or service you use to do so?

**Comment by timur-friedman** [comment]
I use the Mac app version of W3C's service: https://habilis.net/validator-sac/

**Comment by timur-friedman** [comment]
See also: https://validator.w3.org/source/

**Comment by timur-friedman** [comment]
I can't figure out why these differences exist and whether these additional error notifications are useful  or not. I'll happily be guided by you, @bsenel , regarding which validator to use.

**Comment by bsenel** [comment]
According to my understanding, https://habilis.net/validator-sac/ is a service based on W3C's Validator, but as they mention on their website, this is a version. Additionally, when I check the download section, I see the tool covers Mac OS X 10.7 to 10.11, which makes me feel this tool is not up-to-date since my Mac OS X is 10.15.4.

If you look at https://validator.w3.org/source/, in the very first paragraph, they steer us directly to https://validator.w3.org/nu/. When I use that for validation, for example, https://validator.w3.org/nu/?doc=http%3A%2F%2Fwww.edge-net.org, I don't encounter errors but a warning.

So, long story short, I would prefer to stick on https://validator.w3.org/ and close these issues, @timur-friedman, since there aren't any errors.

**Comment by chuckhoupt** [comment]
Hi, author of Validator-SAC here. Validator-SAC does have both the classic Validator and Nu-Validator built-in, but as you note, it is currently out-of-date. This means that it doesn't have the latest list of allowed `link-rel` values. So when you see a difference, always to defer to https://validator.w3.org.

**Comment by bsenel** [comment]
Hi! Thank you for visiting us and providing the information. That really saves us to spend more time on this issue. So, I close the issue according to the advice provided by the author of Validator-SAC.
